BioArchLinux/spades/PKGBUILD

```diff
@@ -9,84 +7,34 @@ pkgrel=1
 pkgdesc='SPAdes Genome Assembler'
 url='http://cab.spbu.ru/software/spades/'
 arch=('x86_64')
-license=('GPLv2')
-depends=('python')
+license=('GPL2')
+depends=(
+  python
+)
 makedepends=(
-    'gcc'
-    'cmake'
-    'make'
-    'pkgconf'
-    'zlib'
-    'bzip2'
-    'bash'
-    'openmp'
+  bzip2
+  cmake
+  openmp
+  zlib
 )
-checkdepends=('python')
+options=("!lto")
 
 source=(
-    "https://github.com/ablab/spades/archive/refs/tags/v$pkgver.tar.gz"
-    "spades"
+  "${pkgname}-${pkgver}.tar.gz::https://github.com/ablab/spades/archive/refs/tags/v${pkgver}.tar.gz"
 )
-sha512sums=('30aed03cc6047b10657a850897fc43058c0e2deef5402081278b91442820d0829c02505357c84c3dd809ae41adf315c7da195027c5a58fdd9ce9a343e0049879'
-            '20fa67151bea016e088058171cc1cb8668b810046e859a16b609711d7a1e539d4ddb37835bd5c9b881696f6ff9eaec5f0d40873ed09359c100687b861f87d215')
-
-prepare() {
-    cd "$srcdir/$pkgname-$pkgver/assembler" || exit
-
-    mkdir -p bin
-}
+sha512sums=('30aed03cc6047b10657a850897fc43058c0e2deef5402081278b91442820d0829c02505357c84c3dd809ae41adf315c7da195027c5a58fdd9ce9a343e0049879')
 
 build() {
-    cd "$srcdir/$pkgname-$pkgver/assembler" || exit
-
-    ./spades_compile.sh
-}
-
-check() {
-    cd "$srcdir/$pkgname-$pkgver/assembler/bin/" || exit
-
-    python spades.py --test
+  cmake -B ${pkgname}-${pkgver}/assembler/build_spades -S ${pkgname}-${pkgver}/assembler/src \
```

Comment:
Great!

Reply:
I should change `${pkgname}-${pkgver}/assembler/build_spades` to `build`. The `python spades --test` only works if we use the default `PREFIX`.